<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1187</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="profile" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>-30</y>
      <width>1131</width>
      <height>571</height>
     </rect>
    </property>
    <widget class="QLabel" name="profileTitleLabel">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>40</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Profile</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="friendTableWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>181</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="addFriendButton">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>60</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Friend</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="suggestionsTableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>291</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>1030</x>
       <y>500</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>back</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="postTextEdit">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>460</y>
       <width>421</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>340</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Friend Suggestions</string>
     </property>
    </widget>
    <widget class="QLabel" name="friend_status">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>60</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Friend Status</string>
     </property>
    </widget>
    <widget class="QPushButton" name="uploadProfile">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>290</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Upload</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="BioTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>291</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="SaveBio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>500</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Bio</string>
     </property>
    </widget>
    <widget class="QPushButton" name="OpenMoodboard">
     <property name="geometry">
      <rect>
       <x>1030</x>
       <y>70</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Mood</string>
     </property>
    </widget>
    <widget class="QLabel" name="profilePicLabel">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>130</y>
       <width>151</width>
       <height>141</height>
      </rect>
     </property>
     <property name="text">
      <string>Pic Label</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="postsScrollArea">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>100</y>
       <width>551</width>
       <height>341</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>549</width>
        <height>339</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="postButton">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>530</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Post</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="login_2" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>29</y>
      <width>341</width>
      <height>131</height>
     </rect>
    </property>
    <widget class="QTextEdit" name="login">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>231</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="myButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter your Name</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="bioLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>510</y>
      <width>571</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>biography</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1187</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
